index.d.ts

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
We should probably change this to single-quotes to match the code-style of the rest of the project.